examples/addons/http-trailers.py

```diff
@@ -15,14 +15,29 @@ def request(flow: http.HTTPFlow):
     if flow.request.trailers:
         print("HTTP Trailers detected! Request contains:", flow.request.trailers)
 
+    if flow.request.path == "/inject_trailers":
+        if not flow.request.is_http2:
```

Comment:
With HTTP/3 being on the horizon, I would prefer making this an explicit check for HTTP/1.1.
And does HTTP/1.0 support trailers at all? Should we restrict it to HTTP/1.1?
Please add a new `is_http11` function similar to the existing `is_http2` one to `mitmproxy/net/http/message.py`.

Reply:
Chunked transfer encoding was introduced in 1.1, so yes this won't work in 1.0. I'll look at adding an `is_http11` function.

Reply:
Added both `is_http10` and `is_http11`.  mitmproxy does not appear to support HTTP/0.9, so I didn not include it.